subsys/net/ip/ipv6.h

```diff
@@ -205,7 +205,11 @@ static inline int net_ipv6_finalize(struct net_pkt *pkt,
 #if defined(CONFIG_NET_IPV6_MLD)
 int net_ipv6_mld_join(struct net_if *iface, const struct in6_addr *addr);
 #else
-#define net_ipv6_mld_join(...)
+static inline int
+net_ipv6_mld_join(struct net_if *iface, const struct in6_addr *addr)
+{
```

Comment:
The arguments should be explicitly marked as unused with `ARG_UNUSED()` macro.

Reply:
Right.